### ISSUE TITLE
iOS 9.0.2 iPhone not working

### ISSUE THREAD

**Issue by kishorekumaru** [gbenvenuti/cordova-plugin-screen-orientation#73]
To run phonegap app I am using phonegap emulator for iPhone and iPad, 
I am getting screen.orientation as undefined.  
cordova version : 5.3.3
phonegap  version : 5.2.0
iPhone 6(iOS 9.0.2) - phonegap emulator console is showing Orientation is undefined
iPad (iOS 9.0.2) - Same as iPhone

**Comment by gbenvenuti** [comment]
Hi @kishorekumaru 

This issue is not specific to iOS9 it appears javascript related, either:

* the plugin did not install correctly OR
* the call is being made outside of `deviceready`

There are a previous issues similar to this.

**Comment by kishorekumaru** [comment]
I created a new phonegap project from scratch installed the plugin, since my cordova version is 5.3.3 I have used cordova plugin add cordova-plugin-screen-orientation

1. This is my fetch.json

`{
    "cordova-plugin-screen-orientation": {
        "source": {
            "type": "registry",
            "id": "cordova-plugin-screen-orientation"
        },
        "is_top_level": true,
        "variables": {}
    }
}`

2. This is my `deviceready` function

```onDeviceReady: function() {    app.receivedEvent('deviceready'); },
receivedEvent: function(id) {
   var parentElement = document.getElementById(id);
   var listeningElement = parentElement.querySelector('.listening');
   var receivedElement = parentElement.querySelector('.received');
  listeningElement.setAttribute('style', 'display:none;');
   receivedElement.setAttribute('style', 'display:block;');
   // screen.lockOrientation('landscape');
   console.log('Orientation is ' + screen.orientation);
   console.log('Received Event: ' + id);
}```


My Phonegap console
*[phonegap] starting app server...
[phonegap] listening on 192.168.12.172:3000
[phonegap]
[phonegap] ctrl-c to stop the server
[phonegap]
[phonegap] 200 /__api__/appzip
[phonegap] 200 /socket.io/socket.io.js
[phonegap] 200 /socket.io/?EIO=2&transport=polling&t=1444788168447-0
[phonegap] 200 /socket.io/?EIO=2&transport=polling&t=1444788168508-1&sid=HETrK5M1l3n0aixiAAAA
[phonegap] [console.log] __Orientation is undefined__
[phonegap] 200 /socket.io/?EIO=2&transport=polling&t=1444788168534-2&sid=HETrK5M1l3n0aixiAAAA
[phonegap] [console.log] Received Event: deviceready
[phonegap] 200 /socket.io/?EIO=2&transport=polling&t=1444788168572-4&sid=HETrK5M1l3n0aixiAAAA
[phonegap] 200 /socket.io/?EIO=2&transport=polling&t=1444788168546-3&sid=HETrK5M1l3n0aixiAAAA*


Please let me know if I miss something,

**Comment by joelcdoyle** [comment]
+1 see my comments on issue [#67](https://github.com/gbenvenuti/cordova-plugin-screen-orientation/issues/67)

**Comment by tony--** [comment]
@kishorekumaru - @gbenvenuti has released 1.4.2 which should fix this issue. Will you try 1.4.2 and provide an update, please?

**Comment by tony--** [comment]
AFAIK this issue is resolved so I'm going to close this.
OP or other commenters - please provide additional feedback if this issue is not resolved for you.
